test/features/deploy_service_resources.feature

```diff
@@ -0,0 +1,33 @@
+@resoures
+Feature: Deploy the service by configuring the resource requests and limits
+
+  Background:
+    Given Namespace is created
+    And Kogito Operator is deployed
+
+  @runtimerequests @runtimelimits
+  Scenario Outline: Setting runtime resource "requests" and "limits"
+    Given Clone Kogito examples into local directory
+    And Local example service "ruleunit-quarkus-example" is built by Maven
+
+    When Create service "ruleunit-quarkus-example" with runtime requests <requests> and limits <limits>
```

Comment:
well, datatable would cause multiple test execution
the way it is implemented now we can test both requests and limits at the same time

Reply:
> well, datatable would cause multiple test execution

it won't in fact.
I just want to replace:
`Create service "ruleunit-quarkus-example" with runtime requests <requests> and limits <limits>`
with:
```
Create service "ruleunit-quarkus-example" with runtime resources:
  | requests     | <requests> |
  | limits       | <limits> |
```
If just find it easier to read.

And the examples datatable will stay as it is

Reply:
oh, ok, in that case sounds good to me

Reply:
Okis, will do. I like this :)